datasets/turk/README.md

```diff
@@ -0,0 +1,142 @@
+---
+title: 'TURKCorpus'
+type: Simplification
+motivation: TURKCorpus is a high-quality simplification dataset where each source sentence is associated with 8 human-written simplifications.
+---
+
+# Dataset Card for TURK
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** None 
+- **Repository:** [TURKCorpus](https://github.com/cocoxu/simplification)
+- **Paper:** [Optimizing Statistical Machine Translation for Text Simplification](https://www.aclweb.org/anthology/Q16-1029/)
+- **Leaderboard:** N/A
+- **Point of Contact:** [Wei Xu](mailto:wei.xu@cc.gatech.edu)
+
+
+### Dataset Summary
+
+TURKCorpus is a multi-reference dataset for the evaluation of sentence simplification in English. The dataset consists of 2,359 sentences from the [Parallel Wikipedia Simplification (PWKP) corpus](https://www.aclweb.org/anthology/C10-1152/). Each sentence is associated with 8 crowdsourced simplifications that focus on only lexical paraphrasing (no sentence splitting or deletion).
+
+### Supported Tasks and Leaderboards
+
+No Leaderboard for the task.
+
+### Languages
+
+TURKCorpus contains English text only (BCP-47: `en`).
+
+## Dataset Structure
+
+### Data Instances
+
+- `simplification` configuration: an instance consists of an original sentence and 8 possible reference simplifications that focus on lexical paraphrasing.
```

Comment:
Here we expect an example from the actual dataset object.
You can get one by using `load_dataset` and then taking the first sample `dataset["train"][0]` for example.